src/reducers/articleReducers/ArticleReducer.spec.js

```diff
@@ -0,0 +1,118 @@
+import ArticleReducer from "./ArticleReducer";
```

Comment:
reducer tests should also be placed in the reducer `__tests__` folder, that is the convention we agreed to.